src/github_utils.py

```diff
@@ -19,111 +20,104 @@ def get_pull_request_diff():
     # Get diff content
     diff_url = pull_request.diff_url
     response = requests.get(diff_url)
```

Comment:
Added type hints for clarity.
```suggestion
|
```
N/A